jenkins/internal-ci/generic/pr/cortx_deployment/cortx-rgw.groovy

```diff
@@ -0,0 +1,309 @@
+pipeline { 
+    agent {
+        node {
+            label "ceph-build-node"
+        }
+    }
+
+    options { 
+        skipDefaultCheckout()
+        timeout(time: 180, unit: 'MINUTES')
+        timestamps()
+        disableConcurrentBuilds()
+        ansiColor('xterm')  
+    }
+
+    parameters {  
+	    string(name: 'RGW_URL', defaultValue: 'https://github.com/Seagate/cortx-rgw', description: 'Repo for rgw')
+        string(name: 'RGW_BRANCH', defaultValue: 'main', description: 'Branch for rgw')
+        choice(name: 'DEPLOY_BUILD_ON_NODES', choices: ["Both", "1node", "3node" ], description: '''<pre>If you select Both then build will be deploy on 1 node as well as 3 node. If you select 1 node then build will be deploy on 1 node only. If you select 3 node then build will be deploy on 3 node only. 
+</pre>''')
+        string(name: 'CORTX_RE_URL', defaultValue: 'https://github.com/Seagate/cortx-re', description: 'Repo for cortx-re')
+        string(name: 'CORTX_RE_BRANCH', defaultValue: 'main', description: 'Branch for cortx-re')
+
+        choice (
+            choices: ['all', 'cortx-all', 'cortx-control', 'cortx-rgw', 'cortx-data'],
```

Comment:
Remove `cortx-all` image

Reply:
Ok

Reply:
Done.

Reply:
@gauravchaudhari02  - Can you please check this. Ideally, we should not be using `cortx-all` images right

Reply:
Removed it